app/views/harvests/index.html.haml

```diff
@@ -1,4 +1,12 @@
-- content_for :title, @owner ? t('.title.owner_harvests', owner: @owner) : @crop ? t('.title.crop_harvests', crop: @crop.name) : t('.title.default')
+- title = if @owner
+  = t('.title.owner_harvests', owner: @owner)
```

Comment:
Should this really be an `=`? We want to return the value Ruby-side, not interpolate it into the generated HTML.

More generally, could we make this code a helper? It seems complex enough to be worth it.